packages/web-components/src/components/content-block-segmented/__stories__/content-block-segmented.stories.react.tsx

```diff
@@ -0,0 +1,265 @@
+    <DDSImageItem media="(min-width: 320px)" srcset={imgSm16x9} />
+  </DDSImage>
+);
+
+const contentItemCopy =
+  'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus ' +
+  'sed interdum tortor. Sed id pellentesque diam. In ut quam id mauris finibus ' +
+  'efficitur quis ut arcu. Praesent purus turpis, venenatis eget odio et, ' +
+  'tincidunt bibendum sem. Curabitur pretium elit non blandit lobortis. Donec ' +
+  'quis pretium odio, in dignissim sapien. Lorem ipsum dolor sit amet, ' +
+  'consectetur adipiscing elit. Vivamus sed interdum tortor. Sed id pellentesque ' +
+  'diam. In ut quam id mauris finibus efficitur quis ut arcu. Praesent purus ' +
+  'turpis, venenatis eget odio et, tincidunt bibendum sem. Curabitur pretium ' +
+  'elit non blandit lobortis. Donec quis pretium odio, in dignissim sapien.';
+
+const video = <DDSVideoPlayerContainer slot="media" video-id="0_uka1msg4"></DDSVideoPlayerContainer>;
+
+const linkListItems = [
+  'Containerization A Complete Guide',
+  'Why should you use microservices and containers',
+  'Learn more about Kubernetes',
+  'Explore AI use cases in all industries',
+];
+
+export const Default = args => {
+  const { heading, copy, ctaStyle, ctaType, complementaryStyleScheme } = args?.ContentBlockSegmented ?? {};
+  return (
+    <DDSContentBlockSegmented complementary-style-scheme={complementaryStyleScheme}>
+      <DDSContentBlockHeading>{heading}</DDSContentBlockHeading>
+      <DDSContentBlockCopy>{copy}</DDSContentBlockCopy>
+      {image}
+      <DDSContentBlockSegmentedItem>
+        <DDSContentGroupHeading>Lorem ipsum dolor sit amet.</DDSContentGroupHeading>
+        <DDSContentItemCopy>{contentItemCopy}</DDSContentItemCopy>
+        <DDSTextCTA slot="footer" cta-type="local" icon-placement="right" href="https://example.com">
+          Lorem Ipsum dolor sit
+        </DDSTextCTA>
+      </DDSContentBlockSegmentedItem>
+      <DDSContentBlockSegmentedItem>
+        <DDSContentGroupHeading>Lorem ipsum dolor sit amet.</DDSContentGroupHeading>
+        <DDSContentItemCopy>{contentItemCopy}</DDSContentItemCopy>
+        {video}
+        <DDSTextCTA slot="footer" cta-type="local" icon-placement="right" href="https://example.com">
+          Lorem Ipsum dolor sit
+        </DDSTextCTA>
+      </DDSContentBlockSegmentedItem>
+      {ctaStyle === 'text' ? (
+        <DDSTextCTA slot="footer" cta-type={ctaType} icon-placement="right" href={hrefsForType[ctaType]}>
+          Lorem ipsum dolor
+        </DDSTextCTA>
+      ) : (
+        <DDSCardLinkCTA slot="footer" cta-type={ctaType} href={hrefsForType[ctaType]}>
+          <DDSCardLinkHeading>Lorem ipsum dolor</DDSCardLinkHeading>
+          <DDSCardCTAFooter />
+        </DDSCardLinkCTA>
+      )}
+    </DDSContentBlockSegmented>
+  );
+};
+
+export const withLinkList = args => {
+  const { blockHeading, heading, copy, ctaStyle, ctaType, complementaryStyleScheme, totalLinks } =
+    args?.ContentBlockSegmented ?? {};
```

Comment:
```suggestion
  const { blockHeading, heading, copy, ctaStyle, ctaType, complementaryStyleScheme, totalLinks } =
    args?.ContentBlockSegmented ?? {};
    document.querySelector('dds-content-block-heading') = blockHeading;
```

Reply:
@emyarod I knew there had been a similar issue for other components where the shadow root prevents the knob from updating. This is the workaround I had previously used, and I'm now using it for the Leadspace with search story.

https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/9407/files#diff-4ac2bb3580c248c5605af672a5a16fc781257362cc62c555ce8f45d4788325d0R41-R45

Reply:
added to the react wrapper and web components stories!